Use Pod::Usage in nagios_zonecheck.pl

bonki opened the pull request:
`nagios_zonecheck.pl` uses `pod2usage()` without importing `Pod::Usage`.

Commit:
Merge pull request #2 from dotse/master

merge from dotse

Commit:
set executable flag to .pl files

stasic committed:
[fix] doc NSEC3

Commit:
Merge pull request #3 from stasic/patch-3

[fix] doc NSEC3

stasic committed:
[fix] add DNSSEC algorithms

added DNSSEC algorithms

Commit:
Merge pull request #4 from stasic/patch-4

[fix] add DNSSEC algorithms

stasic committed:
[fix] sprintf takes only 3 out of 4 arguments

[fix] formatstring in sprintf has only 3 arguments, but 4 were supplied
added the missing fourth argument in formatstring
how to reproduce it:
./dnssec_monitor.pl --debug  --zone rhybar.cz sld1.ns.nic.cz

Commit:
Merge pull request #5 from stasic/patch-5

[fix] sprintf takes only 3 out of 4 arguments

stasic commented:
make your pull request toward https://github.com/pawal/dnssec-monitor (the original repository)

stasic committed:
[fix] use zonename more consistantly and show defaults

3 minor fixes:
- in synopsis aligned long description *zonename* with short description *zone*
- display critical and warning defaults 
- indentation fix up

Commit:
Merge pull request #6 from stasic/patch-1

[fix] use zonename more consistantly and show defaults

bonki committed:
Import Pod::Usage where required

nagios_zonecheck.pl uses pod2usage() without importing Pod::Usage first.

bonki commented:
Sure, thanks.

bonki commented:
This is now merged into upstream.
